src/index.js

```diff
@@ -90,20 +90,21 @@ export function ServerHttpArchive({ har }) {
 }
 
 export function withFetchHar(
-  App,
+  AppOrPage,
   {
     fetch: baseFetch = global.fetch,
     enabled = process.env.NODE_ENV !== "production",
     ...options
   } = {}
 ) {
-  const name = App.displayName || App.name || "App";
+  const name = AppOrPage.displayName || AppOrPage.name || "AppOrPage";
 
   class WithFetchHar extends React.Component {
-    static displayName = `withFetchHar(${App})`;
+    static displayName = `withFetchHar(${AppOrPage})`;
```

Comment:
omg this highlights an existing bug! should be `${name}` here not the passed in component. I'm gonna merge this and just fix in master since it's existing